### PR TITLE
feat added basic routes

### PR DESCRIPTION
**Pull request by Himanshu-Dhawale**
EmpireScreenShot1
EmpireScreenshot 2
EmpireScreenshot 3
EmpireScreenshot 4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,25 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import {Routes, Route} from "react-router-dom";
+import {Navbar} from "./components/Navbar/Navbar";
+import {Cart} from "./pages/Cart/Cart";
+import {Homepage} from "./pages/Homepage/Homepage";
+import {Login} from "./pages/Login/Login";
+import {SignUp} from "./pages/SignUp/SignUp";
+import {Wishlist} from "./pages/Wishlist/WishList";
```
Comment (ApurvaSawant11) [inline src/App.js]: You can create an `index.js` file in the `pages` folder and export all the pages there. That way you can import them in one line

Example:

In `index.js` of pages folder:

export { Homepage } from "./Homepage/Homepage"
export { Login } from "./Login/Login"

And in `App.js`:

import { Homepage, Login,....and so on } from "./pages"


You can do the same for `components` folder in future when you have more components

**Review by ApurvaSawant11 — APPROVED** [review]
The code looks we structured.

Just gave a suggestion. Try it out. It will be easier to import and the code will look cleaner.

File: src/App.js (hunk 1 above)
Comment (Himanshu-Dhawale) [reply src/App.js]: Great Idea. I'll try to implement it

**Comment by Dev-AkshatGupta** [comment]
Code looks good to me and you should follow the suggestions as suggested by Apoorva

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { Link } from 'react-router-dom'
+export const Navbar = () => {
+  return (
+    <div>
+    <nav class="nav-bar">
+    <div class="nav-title">Title</div>
+    <button class="nav-icon"><Link to="/Login"><i class='bx bxs-user'></i></Link></button>
+    <button class="nav-icon"><Link to="/Wishlist"><i class='bx bxs-heart'></i></Link></button>
+    <button class="nav-icon"><Link to="/Cart"><i class='bx bxs-cart'></i></Link></button>
+    </nav>
```
Comment (Maianki) [inline src/components/Navbar/Navbar.jsx]: change class to className.

**Review by Maianki — APPROVED** [review]
Everything looks good just has added one change.

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (Himanshu-Dhawale) [reply src/components/Navbar/Navbar.jsx]: Sure

**Pull request merged**